phy_tester/main/cmd_ethernet.c

```diff
@@ -276,8 +344,7 @@ static int nearend_loopback_enable(int argc, char **argv)
         }
     }
     if (phy_id >= PHY_ID_END) {
-        ESP_LOGE(TAG, "unsupported PHY");
-        return 1;
+        ESP_LOGW(TAG, "untested PHY is used");
```

Comment:
is it intentional to continue here?

Reply:
Yes, if I used "Generic PHY" driver with any PHY, it can work but that particular PHY is untested. If the PHY fully follows the standard , loopback will be set correctly and everything is easy. If doesn't, user was warned and they need to start there.